andela/ah-kgl-avengers-backend
# #165020207 creates, updates, views and deletes articles

## What does this PR do?
Create, updates, views, and deletes an article

## Description of Task to be completed?
Users should be able to create articles

## How should this be manually tested?
Using an API client like Postman or Insomnia:
- To test the create article endpoint Make a POST request to `localhost:3000/api/v1/articles` with a JSON body that includes something like this:
``{
	"title": "Working Now with Aaron",
	"body": "I do not know",
	"author":"11446897-53cf-405d-8c55-89c9891d89cf"
}, ``
if the author is signed up already you should receive a responde of `200` and get this response object:
``{
    "status": 201,
    "article": {
        "id": "4e9ec888-23ec-431f-904b-51c5a60e745d",
        "title": "Working Now with Aaron",
        "description": "I do not know",
        "body": "I do not know",
        "slug": "working-now-with-aar95dcc89fe7",
        "tags": [
            null
        ]
    }
}``
if some fields are wrongly entered you will receive error messages according to the type of error.
# For the update:
you will have to update the title and body
# for viewing one article.
You will have to add the created slug on the route to select that article like this:
localhost:3000/api/v1/articles/i-me-like-bobo-so-mu77e90d002e

## Any background context you want to provide?

## What are the relevant pivotal tracker stories?
[Users should be able to create articles #165020207](https://www.pivotaltracker.com/story/show/165020207)
## Screenshots (if appropriate)
N/A
![image](https://user-images.githubusercontent.com/33352484/56288675-88e37800-611f-11e9-84a2-b4d67e7d69e7.png)

## Questions:
N/A
